chromium_src/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "../../../../../../../chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.h"
+
+#define BRAVE_SHORT_CIRCUIT_TO_FALSE \
```

Comment:
macro name should be `BRAVE_SHOULD_SHOW_SIGNIN_PROMO_`

Reply:
Fixed.